### PR TITLE
Add Sagrada

### PR DESCRIPTION
**Pull request by haidao0923**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -273,6 +273,16 @@
...
 
+### [Sagrada](https://boardgamegeek.com/boardgame/199561/sagrada)
+
+> Each player builds a stained glass window by building up a grid of dice on their player board. Each board has some restrictions on which color or shade (value) of die can be placed there. Dice of the same shade or color may never be placed next to each other. Dice are drafted in player order, with the start player rotating each round, snaking back around after the last player drafts two dice. Scoring is variable per game based on achieving various patterns and varieties of placement...as well as bonus points for dark shades of a particular hidden goal color.
+
+![Sagrada game image](https://cf.geekdo-images.com/PZt3EAAGV3dFIVuwMR0AEw__itemrep/img/fLGFtsAiZgj3VwTDvgCn2yBIDYw=/fit-in/246x300/filters:strip_icc()/pic3525224.jpg)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   1 - 4 |        8 | 30-45m |
```
Comment (edm00se) [inline readme.md:284]:
```suggestion
|   1 - 4 |       10 | 30-45m |
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Thank you for the submission! This looks well formed and passes all the checks. I noticed there's a bit of confusion on the minimum age; specifically in that while BGG lists 14+ (community rating of 8+) [the game publisher, Floodgate Games, cites it as for ages 10+](https://floodgate.games/products/sagrada). If you could update that value I'd love to merge this in.

**Comment by haidao0923** [comment]
From experience, I think that 8 is enough to play the game since the rule is not the complicated. However, I have committed the changes. Let me know if the changes was updated for you as I am still new to this.

**Review by edm00se — APPROVED** [review]

**Comment by edm00se** [comment]
The changes look good, thanks for contributing!

My 8yo can play some games better  than adults, so while age may be subjective, it's likely best to stick to the publisher's listed value, as really the table is a summary from what should be on the box.

**Pull request merged**